proof/access-control/AARCH64/ArchADT_AC.thy

```diff
@@ -8,40 +8,37 @@ theory ArchADT_AC
 imports ADT_AC
 begin
 
-context Arch begin global_naming RISCV64
+context Arch begin global_naming AARCH64
 
 named_theorems ADT_AC_assms
 
-lemma mask_ptTranslationBits_ucast_ucast:
-  "(asid && mask ptTranslationBits) = ucast (ucast asid :: 9 word)"
-  by (word_eqI_solve simp: ptTranslationBits_def)
-
 lemma ptr_offset_in_ptr_range:
-  "\<lbrakk> invs s; x \<notin> kernel_mappings;
-     get_vspace_of_thread (kheap s) (arch_state s) tcb \<noteq> global_pt s;
+  "\<lbrakk> invs s; get_vspace_of_thread (kheap s) (arch_state s) tcb \<noteq> global_pt s;
      get_page_info (aobjs_of s)
-                   (get_vspace_of_thread (kheap s) (arch_state s) tcb) x = Some (base, sz, attr, r) \<rbrakk>
+       (get_vspace_of_thread (kheap s) (arch_state s) tcb) x = Some (base, sz, attr, r) \<rbrakk>
      \<Longrightarrow> ptrFromPAddr base + (x && mask sz) \<in> ptr_range (ptrFromPAddr base) sz"
```

Comment:
I'm a bit surprised this lemma is in Access and not in AInvs, but I guess it's only needed up here.